### DISCUSSION TITLE
Opinion labeling and cites labeling in a search of opinions

### DISCUSSION THREAD

**Discussion by sethchandler** [freelawproject/courtlistener#4400] [Q&A]
Suppose I do a search of opinions using an api call such as something like this.  https://www.courtlistener.com/api/rest/v3/search/?q=&type=o&order_by=score%20desc&stat_Precedential=on&court=tex&filed_after=01%2F01%2F2020.  Ultimately with a little Python I create a dataframe that has columns 'id' and 'cites' and 'cluster_id' among other things. The 'cites' value is a list. But what do they represent? Do they represent the cluster_id of the cases that were cited or the opinion_id of the cases that were cited? 

One reason I ask is that sometimes it appears as if the cites are all stuffed into a particular opinion, i.e. even if Justice X wrote the majority opinion with citations and Justice Y wrote a dissent or concurrence with citations, the cites are all listed as belonging to the opinion written by Justice X.  I have not yet found examples where in a multi-opinion cluster the cites are distributed over the multiple opinions. So, if I wanted to develop a network of citations, I might need to go cluster_id to cluster_id rather than opinion to opinion. Any help appreciated.

**Comment by mlissner** [comment]
Seth, sorry for the slow reply (Labor Day!). Did you see the citation network API docs: https://www.courtlistener.com/help/api/rest/v3/citations/

I think they should really help.
